build_winapp/appxmanifest.xml

```diff
@@ -31,7 +31,8 @@
     <Application
       Id="EndlessOSFoundation.EndlessKey"
       Executable="kolibri-electron.exe"
-      EntryPoint="Windows.FullTrustApplication">
+      EntryPoint="Windows.FullTrustApplication"
+      xmlns="urn:schemas-microsoft-com:asm.v3">
```

Comment:
```
MakeAppx : error: Error info: error C00CE014: App manifest validation error: The app manifest must be valid as per schema: Line 31, Column 6, Note: The schema for MaxVersionTested specified does not recognize XML fields with namespace "urn:schemas-microsoft-com:asm.v3". Please ensure that you have the correct MaxVersionTested specified. Reason: Element '{urn:schemas-microsoft-com:asm.v3}Application' is unexpected according to content model of parent element '{http://schemas.microsoft.com/appx/manifest/foundation/windows10}Applications'.
```

Reply:
Ya!  Trying to find out workaround to tell appcert that kolibri-electron.exe has DPI awareness feature.  But, seems this way is wrong.